frameworks/FSharp/giraffe/src/App/Program.fs

```diff
@@ -0,0 +1,72 @@
+module App.App
+
+open System
+open Microsoft.AspNetCore.Builder
+open Microsoft.AspNetCore.Hosting
+open Microsoft.Extensions.Logging
+open Microsoft.Extensions.DependencyInjection
+open Giraffe
+open Microsoft.AspNetCore.Http
+
+// ---------------------------------
+// Models
+// ---------------------------------
+[<CLIMutable>]
+type JsonMessage = { message : string }
+
+// ---------------------------------
+// Web app
+// ---------------------------------
+
+let jsonUtf8 (data:obj) : HttpHandler =
+    fun (next : HttpFunc) (ctx : HttpContext) ->
+        let bytes = Utf8Json.JsonSerializer.Serialize(data)
+        ctx.SetContentType "application/json"

```

Comment:
Additionally set the content length (bytes.Length); otherwise it will go as chunked encoding which is slightly larger and a little more involved.